src/utils/uriTranslator.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import "strings"
+
+type uriTranslator struct {
```

Comment:
The name is confusing, it isn't used for translation, but for replacement.

Reply:
Renamed to UriConverter